include/circt-c/Dialect/OM.h

```diff
@@ -152,6 +152,14 @@ MLIR_CAPI_EXPORTED bool omAttrIsAReferenceAttr(MlirAttribute attr);
 
 MLIR_CAPI_EXPORTED MlirAttribute omReferenceAttrGetInnerRef(MlirAttribute attr);
 
+//===----------------------------------------------------------------------===//
+// IntegerAttr API
+//===----------------------------------------------------------------------===//
+
+MLIR_CAPI_EXPORTED bool omAttrIsAIntegerAttr(MlirAttribute attr);
+
+MLIR_CAPI_EXPORTED MlirAttribute omIntegerAttrGetAttr(MlirAttribute attr);
```

Comment:
nit: i think our convention so far is usually to just call it "blahTypeAttrGet", you don't need the final "Attr" in "omIntegerAttrGetAttr".